sdk/core/README.md

```diff
@@ -96,6 +101,8 @@ const options = {
   accessApiKey: 'YOUR API KEY HASH VALUE'
 }
 ```
+`encryptedSeed`- randomly auto generated by [Generate seed](#Generate seed) [more detail exaple](#Initialize from seed) material that used as input for a private key generation. Generated as a part of [register](#Create DID) / [signUp](#Sign up) process and stored as encrypted value in protected vault.
```

Comment:
@Volland same here, the links do not display correctly 
![Uploading image.png…]()

Reply:
typo - `more detailed example`